ocfweb/static/scss/pages/_mirrors.scss

```diff
@@ -0,0 +1,13 @@
+.mirrors-pie-chart {
```

Comment:
this needs a `.page-mirrors` or whatever wrapping the entire CSS file, see the other page-specific CSS files for examples